snmp/datadog_checks/snmp/data/default_profiles/apc-pdu.yaml

```diff
@@ -47,26 +47,7 @@ metrics:
         tag: powernet_r_pdu_outlet_status_outlet_state
         mapping:
           1: outlet_status_on
-          0: outlet_status_off
```

Comment:
Has been fixed in #15765, should we backport to 7.48 ?

Reply:
Yes, let's backport https://github.com/DataDog/integrations-core/pull/15765